src/inference.py

```
@@ -0,0 +1,71 @@
+"""������"""
```

Ruff (0.11.9)

stream did not contain valid UTF-8

(E902)